Makefile

```diff
@@ -28,7 +29,7 @@ licencecheck:
 	set -e ;\
  	restricted=$$(paste -s -d ',' restricted-licences.txt) ;\
  	projects=$$(dep status -f='vendor/{{ .ProjectRoot }} ') ;\
- 	$(BUILD_DIR)/bin/licence-compliance-checker -L error -A -r $$restricted $$projects ;
+ 	$(BUILD_DIR)/bin/licence-compliance-checker -L error -A -r $$restricted -m github.com/spf13/cobra=MIT $$projects ;
```

Comment:
I don't think this is needed. 
I can run this target fine without the extra `-m github.com/spf13/cobra=MIT `
Travis also complains about it, but it actually fails to list the modules probably because GO111MODULE isn't enabled
```
ERRO[0000] Failed to list go module github.com/spf13/cobra: output: , error: exit status 1 
```